src/apps/common/submodules/mediaSelect/views/tabs-layout.html

```diff
@@ -12,6 +12,13 @@
 						{{ i18n.mediaSelect.uploadNewMedia }}
 					</a>
 				</li>
+				{{#if enableTextspeechTab}}
+				<li class="navbar-menu-item">
+					<a href="javascript:void(0);" class="navbar-menu-item-link" data-tab="text_to_speech">
+						{{ i18n.mediaSelect.textToSpeech.tab }}
+					</a>
+				</li>
+				{{/if}}
```

Comment:
It would be good to add some indentation to the `if` block, to improve readability and follow documents formatting.
```suggestion
			{{#if enableTextspeechTab}}
				<li class="navbar-menu-item">
					<a href="javascript:void(0);" class="navbar-menu-item-link" data-tab="text_to_speech">
						{{ i18n.mediaSelect.textToSpeech.tab }}
					</a>
				</li>
			{{/if}}
```